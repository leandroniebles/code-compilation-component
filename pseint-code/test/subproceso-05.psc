SubProceso x<-Booga1(a,b)
  Escribir A;
  Escribir B;
  x<-1;
FinSubproceso

SubProceso x<-Booga2
  x<-2;
FinSubproceso

Proceso PruebaFuncion
  Escribir Booga1("a","b");
  Escribir Booga1("c","d")+10;
  Escribir Booga2;
  Escribir Booga2+20;
  Escribir Booga2;
  Escribir Booga2+20;
FinProceso

  
