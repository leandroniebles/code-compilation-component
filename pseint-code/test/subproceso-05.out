a
b
1
c
d
11
2
22
2
22
